src/App.tsx

```diff
@@ -37,11 +36,21 @@ const useAdaptBackgroundColor = () => {
         secondaryTextColor
       );
     }
-  });
+  }, 2000);
 };
 
 function App() {
   useAdaptBackgroundColor();
+  setTimeout(() => {
```

Comment:
Add `setTimeout` in app render looks not quite right to me. Since this plugin have full access to the top window context, you can use other ways to check if the used css vars are changed or not. Can you find a better solution?

Reply:
I think alternatively you can ready the current required css vars and apply them to the links.

Reply:
how?